app/screens/thread/thread.js

```diff
@@ -86,16 +86,21 @@ class Thread extends PureComponent {
 
     hasRootPost = () => {
         return this.props.postIds.includes(this.props.rootId);
-    }
+    };
```

Comment:
Unrelated note: I didn't know we do this on mobile repo. Should we nake a resolution to have or not have going forward?